modules/installation-azure-config-yaml-simple.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-customizations.adoc
+// * installing/installing_azure/installing-azure-government-region.adoc
+// * installing/installing_azure/installing-azure-network-customizations.adoc
+// * installing/installing_azure/installing-azure-private.adoc
+// * installing/installing_azure/installing-azure-vnet.adoc
+// * installing/installing-restricted-networks-azure-installer-provisioned.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="installation-azure-config-yaml-simple_{context}"]
+= Sample customized install-config.yaml file for Azure
+
+You can customize the `install-config.yaml` file to specify more details about your {product-title} cluster's platform or modify the values of the required parameters.
+
+[IMPORTANT]
+====
+This sample YAML file is provided for reference only. You must obtain your `install-config.yaml` file by using the installation program and modify it.
+For a full list and description of all installation configuration parameters, see _Installation configuration parameters for Azure_.
+====
+
+.Sample `install-config.yaml` file for {azure-short}
+[source,yaml]
+----
+apiVersion: v1 <1>
+baseDomain: example.com
+pullSecret: '{"auths": ...}'
+sshKey: ssh-ed25519 AAAA...
+metadata:
+  name: example-cluster
+controlPlane: <2>
+  hyperthreading: Enabled
+  name: master
+  platform:
+    azure:
+      type: Standard_D8s_v3
+  replicas: 3
+compute: <3>
+- hyperthreading: Enabled
+  name: worker
+  platform:
+    azure:
+      type: Standard_D2s_v3
+  replicas: 3
+networking: <4>
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+platform: <5>
+  azure:
```

Comment:
You should include `baseDomainResourceGroupName` in `installconfig.platform.azure`

```suggestion
  azure:
    baseDomainResourceGroupName: example-basedomain-resource-group
```

This should be the resource group that contains the DNS zone for the `baseDomain`. This field is required in public installs, but we have to make it as `optional` because it is not required in `private` installs.